DataFormats/simulation/src/Stack.cxx

```diff
@@ -365,6 +371,13 @@ void Stack::Print(Int_t iVerbose) const
   }
 }
 
+void Stack::Print(Option_t* option) const
+{
+  Int_t verbose = 0;
+  if ( option ) verbose = 1;
```

Comment:
In the change proposed in my pull request, I have just fixed the problem, that the function Print() could not be called via base class in FairRoot.